packaging_automation/postgres-matrix.yml

```diff
@@ -0,0 +1,12 @@
+name: Postgres Version Matrix
+project_name: citus # alternatives: citus, citus-enterprise,pg-auto-failover, pg-auto-failover-enterprise
```

Comment:
```suggestion
project_name: citus # alternatives: citus, citus-enterprise, pg-auto-failover, pg-auto-failover-enterprise
```

Reply:
Removed since there is already a matrix yml in tests folder.